src/__swaps__/screens/Swap/components/SwapOutputAsset.tsx

```diff
@@ -156,7 +156,7 @@ export function SwapOutputAsset() {
       <Box
         as={Animated.View}
         height="full"
-        padding={{ custom: INPUT_PADDING }}
+        paddingTop={{ custom: INPUT_PADDING }}
```

Comment:
we'll probably need some padding L/R 
<img width="559" alt="Screenshot 2024-06-11 at 3 50 15 PM" src="https://github.com/rainbow-me/rainbow/assets/6284525/ee290df4-dd07-4027-844c-498ee69f2fe8">

Reply:
Forgot to push a file, should be fixed if you pull latest

Reply:
verified